src/app/rundown-view/components/offsetable-piece/offsetable-piece.component.ts

```diff
@@ -33,6 +36,43 @@ export class OffsetablePieceComponent {
   @ViewChild('labelTextElement')
   public labelTextElement: ElementRef<HTMLSpanElement>
 
+  public media: Media | undefined
```

Comment:
Can be shortened to `public media?: Media`

Reply:
Good point, i did that!